scripts/check-release-please.mjs

```diff
@@ -0,0 +1,82 @@
+import { strict as assert } from 'assert';
+import * as childProcess from 'child_process';
+import path from 'path';
+import { readFileSync } from 'fs';
+
+/*
+	Enumerates through
+
+	- lerna packages,
+	- release please's manifest, and
+	- release please's config, and
+
+	makes sure all those lists are in sync. If not, guides you to do the correct thing
+*/
+
+const errors = [];
+const logErrorIf = (condition, errorMessage) => {
+	if (condition) {
+		errors.push(errorMessage);
+	}
+};
+const PROJECT_ROOT = path.resolve();
+const readJson = (path) => {
+	return JSON.parse(readFileSync(path));
+};
+const getProcessOutput = (cmd, args) => {
+	const result = childProcess.spawnSync(cmd, args);
+	assert(!result.error, result.error);
+	return result.stdout.toString('utf8');
+}
+
+const lernaList = JSON.parse(
+getProcessOutput('lerna', ['list', '--json'])
+);
+const packageList = new Map(
```

Comment:
It seems we only use the `relativeLocation` for these packages.
Can we use `Set` instead, this will also be consistent with the other variables

Reply:
Makes sense. Made everything into Set.